content/en/docs/contribute/localization.md

```diff
@@ -88,6 +92,11 @@ You'll need to know the two-letter language code for your language. Consult the
 to find your localization's two-letter language code. For example, the
 two-letter code for Korean is `ko`.
 
+If the language you are starting a localization for is spoken in various places
+with significative differences between the variants, it might make sense to
```

Comment:
Significative means having a meaning and significant means large enough to be noticed. Are we aiming for the latter or the former?
```suggestion
with significative differences between the variants, it might make sense to
```

Reply:
Hi @divya-mohan0209! I meant the latter. Will update this PR!

Reply:
Updated!